consensus/mint-client/src/bin/main.rs

```diff
@@ -363,5 +363,14 @@ fn main() {
                 .write_json(&tx_file_path)
                 .expect("failed writing tx file");
         }
+
+        Commands::CheckSig {
+            signature,
+            hash,
+            pubkey,
+        } => {
+            let result = pubkey.verify(&hash, &signature);
+            println!("Verification result = {:?}", result);
```

Comment:
Consider making this a little bit more descriptive to provide users of the tool a little bit more context, potentially adding some information on the consequences of an invalid signature to the minting process.

```suggestion
            if Err(e) = pubkey.verify(&hash, &signature) {
                println!("The signature provided failed verification with error: {:?}"
            }
            println!("The provided signature has been successfully verified");
```

Reply:
Thanks for the suggest -- as a tool meant for system operators and inclusion in shell scripts, we've opted to keep the dialog to a minimum throughout. We will stick with the style for now, and if we change our minds, then reconsider the responses throughout.